Diary/Resource/Controller/DiaryItemViewController.swift

```diff
@@ -0,0 +1,239 @@
+    private let titleTextView: UITextView = {
+        let textView = UITextView()
+        textView.isScrollEnabled = false
+        textView.setContentCompressionResistancePriority(.required, for: .vertical)
+        textView.font = UIFont.preferredFont(forTextStyle: .title2)
+        textView.textContainerInset = UIEdgeInsets(top: .zero, left: .zero, bottom: .zero, right: .zero)
+        return textView
+    }()
+    
+    private let bodyTextView: UITextView = {
+        let textView = UITextView()
+        textView.font = UIFont.preferredFont(forTextStyle: .body)
+        textView.textContainerInset = UIEdgeInsets(top: .zero, left: .zero, bottom: .zero, right: .zero)
+        return textView
+    }()
+    
+    private lazy var titleHeightConstraint =
+    titleTextView.heightAnchor.constraint(equalToConstant: LayoutConstant.titleTextViewMaxHeight)
+    
+    private var isOversized = false {
+        didSet {
+            guard oldValue != isOversized else { return }
+            titleTextView.isScrollEnabled = isOversized
+            titleHeightConstraint.isActive = !oldValue
+            titleTextView.setNeedsUpdateConstraints()
+        }
+    }
+    
+    override func loadView() {
+        view = UIView(frame: .zero)
+        view.backgroundColor = .systemBackground
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.addSubview(mainStackView)
+        titleTextView.becomeFirstResponder()
+        configureTextView()
+        configureNavigationBar()
+        configureMainStackView()
+        addKeyboardDismissAction()
+        addObserver()
+    }
+    
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        save()
+        diaryItemManager.resetDiaryItem()
+    }
+    
+    private func configureTextView() {
+        titleTextView.delegate = self
+        bodyTextView.delegate = self
+        setPlaceholder(for: titleTextView)
+        setPlaceholder(for: bodyTextView)
+    }
+    
+    private func setPlaceholder(for textView: UITextView) {
+        if textView.text.isEmpty {
+            textView.textColor = .systemGray3
+            textView.text = (textView == titleTextView) ? Placeholder.title : Placeholder.body
+        } else {
+            titleTextView.resignFirstResponder()
+        }
+    }
+    
+    private func configureNavigationBar() {
+        navigationController?.navigationBar.scrollEdgeAppearance = navigationController?.navigationBar.standardAppearance
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(systemName: Namespace.moreImage),
+                                                            style: .plain,
+                                                            target: self,
+                                                            action: #selector(showActionSheet))
+        
+        let date = Date()
+        title = (title == nil ? DateFormatterManager().formatDate(date) : title)
```

Comment:
```suggestion
        title = title ?? DateFormatterManager().formatDate(date)
```
로 바꾸는게 더 가독성이 좋아보이네요.